src/pynwb/misc.py

```diff
@@ -231,7 +231,7 @@ def __init__(self, **kwargs):
              'doc': 'the index of the unit in unit_ids to retrieve spike times for'})
     def get_unit_spike_times(self, **kwargs):
         index = getargs('index', kwargs)
-        return self.__iv.get_vector(index)
+        return np.array(self.__iv.get_vector(index))
```

Comment:
This is okay if referential equality if never required for the return values of this method.  This now creates the property that `self.get_unit_spike_times(i) != self.get_unit_spike_times(i)`. 

This will also duplicate whatever `IndexedVector.get_vector` returns. Assuming we never want `IndexedVector.get_vector` to cache, this probably won't do much to memory footprint, since the GC will pick up the first copy, but it does incur the extra overhead allocating/copying.